code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -220,14 +211,9 @@
 						L.adjust_fire_stacks(damage_per_tick)
 						L.IgniteMob()
 			else if(istype(target,/obj/mecha))
-				var/sending_message = FALSE
-				if(mech_damage_cycle > 1)
-					mech_damage_cycle = 0
-					sending_message = TRUE
-				else
-					mech_damage_cycle++
 				var/obj/mecha/M = target
-				M.take_directional_damage(damage_per_tick, "fire", get_dir(src, M), 0, 1, sending_message)
+				M.take_damage(damage_per_tick, BURN, "melee", 1, get_dir(src, M)) //does about half of standard damage to mechs * whatever their fire armor is
```

Comment:
Also this was using "fire" armor so it'd do more damage, is that preserved at all?